src/interface/src/app/account-validation/account-validation.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../services';
+import { ActivatedRoute } from '@angular/router';
+import { take } from 'rxjs';
+
+@Component({
+  selector: 'app-account-validation',
+  templateUrl: './account-validation.component.html',
+  styleUrls: ['./account-validation.component.scss'],
+})
+export class AccountValidationComponent implements OnInit {
+  protected isValidated$ = false;
```

Comment:
usually `someproperty$` means is observable/async, so this should be `isValidated`